code/pages/ProductPage.php

```diff
@@ -433,6 +433,11 @@ public function hiddenTag($name=null, $val=null) {
 			$val
 		);
 	}
+	
+	// get FoxyCart Store Name for JS call
```

Comment:
Should this check for the cacheable templates module and the setting? Otherwise it'll include the script even if cacheable templates are enabled.

```php
	public function getColorBoxScript() {
		return (!class_exists('FoxyStripeCache_Controller') && !SiteConfig::current_site_config()->CartPage == true)
			? '<script src="//cdn.foxycart.com/' . FoxyCart::getFoxyCartStoreName() . '/foxycart.colorbox.js?ver=2"></script>'
			: false;
	}
```